jlm/rvsdg/bitstring/value-representation.hpp

```diff
@@ -426,7 +426,11 @@ class bitvalue_repr
   ne(const bitvalue_repr & other) const
   {
     if (nbits() != other.nbits())
-      throw jlm::util::error("Unequal number of bits.");
+    {
+      const std::string & msg = "Unequal number of bits in ne, " + std::to_string(nbits())
+                              + " != " + std::to_string(other.nbits());
```

Comment:
Rather use `strfmt`, please:
`jlm::util::strfmt("Unequal number of bits in ne: ", nbits(), " != ", other.nbits());`

See [strfmt.hpp](https://github.com/phate/jlm/blob/master/jlm/util/strfmt.hpp)

Reply:
Resolved in next commit